arch/arm/src/stm32h7/stm32_allocateheap.c

```diff
@@ -44,13 +44,21 @@
 #include "arm_internal.h"
 #include "hardware/stm32_memorymap.h"
 #include "stm32_mpuinit.h"
-#include "stm32_dtcm.h"
+
+#ifdef CONFIG_ARCH_CHIP_STM32H7_CORTEXM7
+#  include "stm32_dtcm.h"
+#endif
 #include "stm32_fmc.h"
 
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
 
+#if defined(CONFIG_ARCH_CHIP_STM32H7_CORTEXM7) && \
+    defined(CONFIG_STM32H7_CORTEXM4_DISABLED)
+
+/* Configuration for M7 core and M4 core support disabled */
```

Comment:
```suggestion
/* Configuration for M7 core when M4 core support disabled */
```

I think that's the intended message?

Reply:
correct :) fixed